src/Searchable.php

```diff
@@ -143,13 +165,31 @@ public function dependentClasses()
     }
 
     /**
+     * Get document type
+     *
      * @return string
      */
     public function getElasticaType()
     {
         return get_class($this->owner);
     }
 
+    /**
+     * Get published status
+     *
+     * @return bool
+     */
+    public function getElasticaPublishedStatus()
+    {
+        $isLive = true;
+        if ($this->owner->hasExtension(Versioned::class)) {
+            if ($this->owner instanceof SiteTree) {
+                $isLive = $this->owner->isPublished();
+            }
+        }
+        return (bool)$isLive;
```

Comment:
newlines - for readability. If you have phpcs-fixer installed, you cpuld just run it over the codebase. It will deal with spacings too.

Reply:
Fixed